NitroxClient/ClientAutoFacRegistrar.cs

```diff
@@ -9,8 +9,6 @@
 using NitroxClient.Communication.MultiplayerSession;
 using NitroxClient.Communication.NetworkingLayer.LiteNetLib;
 using NitroxClient.Communication.Packets.Processors.Abstract;
-using NitroxClient.Debuggers;
-using NitroxClient.Debuggers.Drawer;
```

Comment:
Same here with debug. Would again just insert `Debuggers.` and `Debuggers.Drawer.` in L57 to L71